repl/src/main/resources/fake_shell.py

```diff
@@ -26,6 +26,12 @@
 import base64
 import os
 import re
+import cloudpickle
+import threading
+import tempfile
+import shutil
+from pyspark import HiveContext
```

Comment:
You cannot do this if unit tests are running. They fail for me locally:

```
- should execute `1 + 2` == 3 *** FAILED *** (1 second, 179 milliseconds)
  JObject(List((status,JString(error)), (execution_count,JInt(0)), (ename,JString(Error)), (evalue,JString(Interpreter died:
  Traceback (most recent call last):
    File "/work/cdh/livy/repl/target/tmp/817353885763592142", line 33, in <module>
      from pyspark import HiveContext
  ImportError: No module named pyspark)), (traceback,JObject(List())))) did not equal JObject(List((status,JString(ok)), (execution_count,JInt(0)), (data,JObject(List((text/plain,JString(3))))))) (PythonSessionSpec.scala:42)                                                                            
```

You need to follow other examples in this file where it avoids importing Spark dependencies when unit tests are being run.